api/envoy/extensions/filters/http/kill_request/v3/kill_request.proto

```diff
@@ -19,10 +19,19 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 
 // Configuration for KillRequest filter.
 message KillRequest {
+  // On which direction should the filter check for the `kill_request_header`.
+  // Default to `INBOUND` if unspecified.
+  enum Direction {
+    INBOUND = 0;
+    OUTBOUND = 1;
```

Comment:
Question on whether to call this REQUEST/RESPONSE or DECODE/ENCODE?

Reply:
decode and encode happen in each direction.  Vote for REQUEST/RESPONSE

Reply:
Went with REQUEST/RESPONSE